__device-tests__/helpers/caps.js

```diff
@@ -1,15 +1,27 @@
-exports.ios = {
+const ios = {
 	browserName: '',
 	platformName: 'iOS',
-	platformVersion: '13.0',
-	deviceName: 'iPhone 11 Simulator',
+	platformVersion: '13.3',
```

Comment:
I think that the `platformVersion` and `deviceName` could be removed from here since the `ios` is not exported. They are declared in `iosLocal `and `iosServer`.